src/components/regions/GLAMMark.svelte

```diff
@@ -16,7 +15,7 @@ a {
 }
 </style>
 
-<a href={getPath('/', $currentQuery)} on:click={store.reset}>
+<a href={`/?${$currentQuery}`} on:click={store.reset}>
```

Comment:
This is a separate point but I think this is a bit of an anti-pattern. I'd argue the logo should always do a `href="/"` with the hard refresh and everything. Anyway it's fine within this PR.

Reply:
Sweet. Please open an issue where we can discuss.